packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1953,9 +1953,17 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
     instance.dragStartBehavior = DragStartBehavior.down;
   }
 
+  bool _canHandleScrollGestures() {
+    return enableGestures
+      && _effectiveScrollController != null
+      && _effectiveScrollController!.positions.length == 1
+      && _effectiveScrollController!.position.hasContentDimensions
+      && _effectiveScrollController!.position.maxScrollExtent > 0;
```

Comment:
maxScrollExtent is a double so maybe use "> 0.0"?

Reply:
Sadly (and perhaps it's my fault) this file isn't consistent about using d.0 or just d for doubles. I changed this one.